### ISSUE TITLE
[FEATURE] Request some more menu item in Wiki page

### ISSUE THREAD

**Issue by suhuide** [MarkDing/Silabs-IoT-Quickstart#4]
**Is your feature request related to a problem? If so, please describe.**
Currently,under product line(Zigbee...),only have course for day1
**Describe the new feature you'd like**
Should add some item,like BEGINNER,INTERMEDIATE,ADVANCE

@Jim-tech

**Comment by MarkDing** [comment]
@suhuide I agree that we should have more inherit layer. Day 1 can be the Boot Camp. And following a series of workshop @Jim-tech What do you think?

**Comment by Jim-tech** [comment]
Thanks for the suggestions. I changed the WiKi structure a little bit, could you please take a look at it? @suhuide

**Comment by suhuide** [comment]
It looks good for me,thanks!

**Comment by MarkDing** [comment]
You mean the changes in Zigbee page? I am sorry that I don't see the difference there. Can you point it out for me? @suhuide @Jim-tech

**Comment by Jim-tech** [comment]
@MarkDing The side bar of the wiki page was changed a little bit.
